File: Warmup/Plus_Minus/plus_minus.cpp
#include <bits/stdc++.h>

int main()
{
    int n = 0;
    std::cin >> n;

    double pos = 0.0;
    double neg = 0.0;
    double zero = 0.0;

    int val = 0;
    for (int i = 0; i < n; ++i)
    {
      std::cin >> val;

      val == 0 ? zero++ : val > 0 ? pos++ : neg++;
    }

    std::cout << std::fixed;
    std::cout << std::setprecision(6);
    
    std::cout << (pos / n) << std::endl;
    std::cout << (neg / n) << std::endl;
    std::cout << (zero / n) << std::endl;

    return 0;
}

File: Warmup/Plus_Minus/plus_minus.py
#!/bin/python3

if __name__=="__main__":
    n = int(input())
    arr = list(map(int, input().split()))

    pos = len(list(filter(lambda x : (x > 0), arr)))
    neg = len(list(filter(lambda x : (x < 0), arr)))
    zero = len(arr) - (pos + neg)

    print("%.6f" % (pos/n))
    print("%.6f" % (neg/n))
    print("%.6f" % (zero/n))
